src/test/java/uk/gov/pay/adminusers/resources/InviteResourceServiceCompleteIT.java

```diff
@@ -20,7 +20,7 @@ class InviteResourceServiceCompleteIT extends IntegrationTest {
     public static final String INVITES_RESOURCE_URL = "/v1/api/invites";
 
     @Test
-    void shouldReturn200withDisabledInviteLinkingToCreatedUser_WhenPassedAValidInviteCode() {
+    void shouldReturn200withDisabledInviteLinkingToCreatedUser_WhenPassedAValidInviteCode() throws Exception {
```

Comment:
Could we add a test case to check that a 422 error is returned if the `second_factor` provided isn't an allowed value?

Reply:
It turns out DropWizard (or so I presume) returns a 400 when it tries to match the argument value to the enum and fails, so we can't easily have a 422, but I've added a test to check for the 400.